articles/iot-hub/iot-hub-devguide-direct-methods.md

```diff
@@ -64,6 +66,25 @@ Direct method invocations on a device are HTTP calls which comprise:
     }
 }
 ```
+Following shows an example of a valid request:
+ 
+ ```
+POST /twins/D001/methods?api-version=2016-11-14 HTTP/1.1
+Host: myhub.azure-devices.net
+Authorization: SharedAccessSignature sr=myhub.azure-devices.net&sig=Zg**SbzCyAKrc%2BDc%3D&se=1515923812&skn=iothubowner
```

Comment:
Should we remove "myhub" and the sig? For example:
 
Host: {yourhubname}.azure-devices.net 
Authorization: SharedAccessSignature sr={yourhubname}.azure-devices.net&sig={yoursigvalue}&se=1515923812&skn=iothubowner